steps/identity/appointee-form-download/AppointeeFormDownload.js

```diff
@@ -16,6 +17,22 @@ class AppointeeFormDownload extends Question {
         return this.fields.benefitType.value;
     }
 
+    get formDownload() {
+
+        const benefitType = this.fields.benefitType.value;
+        let formDownload = {};
+
+        if (benefitType === 'Carer’s Allowance' || benefitType === 'Child Benefit') {
```

Comment:
We shouldn't be creating benefits types on the fly, ideally we should reference the benefit types from here: https://github.com/hmcts/submit-your-appeal/blob/master/steps/start/benefit-type/types.js. However, it doesn't help that these are defined in an array. I'm not sure how much work it is to convert the array into an object as the `/benefit-type` step requires an array. Maybe we can have both without duplication.